easybuild/easyconfigs/s/SuperLU/SuperLU-5.1.1-ictce-7.35.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'CMakeMake'
+
+name = 'SuperLU'
+version = '5.1.1'
+
+homepage = 'http://crd-legacy.lbl.gov/~xiaoye/SuperLU/'
+description = """SuperLU is a general purpose library for the direct solution of large, sparse, nonsymmetric systems of linear equations on high performance machines."""
+
+toolchain = {'name': 'ictce', 'version': '7.3.5'}
+toolchainopts = {'opt': True, 'pic': True}
+
+source_urls = ['http://crd-legacy.lbl.gov/~xiaoye/SuperLU/']
+sources = ["superlu_%(version)s.tar.gz"]
+
+# Let's store the checksum in order to be sure it doesn't suddenly change
+checksums = ['260a3cd90b2100122abff38587a8290a']
+
+# Set to ON to generate shared library
+configopts = "-DBUILD_SHARED_LIBS=OFF"
+
+# Make sure not to build the slow BLAS library included in the package
+configopts += " -Denable_blaslib=OFF"
+
+# Set the BLAS library to use
+# cf https://cmake.org/cmake/help/latest/module/FindBLAS.html
+configopts += " -DBLA_VENDOR=Intel10_64lp"
+
+# Make symlink to version-numbered library filename
+installopts = ' && cd %(installdir)s/lib && ln -s  libsuperlu_%(version)s.a libsuperlu.a'
+
+builddependencies = [('CMake', '3.2.3')]
```

Comment:
Doesn't really matter I guess, I just don't see a reason why not to go with the latest.

Up to you...

Let me know, since this is basically ready to go,with working test reports and debunked remarks. ;)